foreman-proxy-content-enable

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set +e
+
+TMP=$(mktemp -t katello-selinux-enable.XXXXXXXXXX)
+trap "rm -rf '$TMP'" EXIT INT TERM
+LOG=/var/log/foreman-selinux-install.log
+
+for selinuxvariant in targeted
+do
+  if /usr/sbin/semodule -s $selinuxvariant -l >/dev/null; then
+    # Load policy
+    /usr/sbin/semanage module -S $selinuxvariant \
+      -a /usr/share/selinux/${selinuxvariant}/foreman-proxy-content.pp.bz2
+
+    # Append to log file
+    echo "$(date) $0" >> $LOG
+    cat $TMP_EXEC_BEFORE >> $LOG
+    cat $TMP_EXEC_AFTER >> $LOG
+
+    # Commit the changes
+    test -s $TMP && /usr/sbin/semanage -S $selinuxvariant -i $TMP | tee -a $LOG
```

Comment:
This is always false. `TMP` is always empty since it's created and never written to
```suggestion
```